README.md

```diff
@@ -1,128 +1,207 @@
+> Due to my insufficient skill, the code is glutted with obscure writings and terrible logic, turning the whole thing into a complete mess. I'm sorry if that disturbs you.  
+
+Ack cannot be contacted at this time, and development has been taken over by others.
+
+---
+
+### My antivirus says thprac is malware. Is it safe?
+Thprac contains no malicious code, but its behavior may trigger your antivirus. If flagged, add an exception or trust rule in your antivirus to allow thcrap to work. Alternatively, you can try [this version](https://github.com/touhouworldcup/thprac/issues/112) and report back if the issue persists.
+
+---
+
+## Compatibility
+
+### Does thprac work with English patches?
+Thprac is compatible with **thcrap** and can function as a thcrap launcher, similar to the [Universal THCRAP Launcher](https://github.com/thpatch/Universal-THCRAP-Launcher/). It does not support static English patches, like those created by gensokyo.org.
+
+---
+
+### thprac cannot find Embodiment of Scarlet Devil. What should I do?
+Ensure your executable is named either `東方紅魔郷.exe` or `th06.exe`. If thprac still fails to detect it, you can manually launch the game with any filename through the launcher, which will attach thprac to the game.
+
+---
+
+## Features
+
+### How do I save a replay mid-game in TH06-10?
+Saving replays mid-game is not supported due to technical limitations. As per the thprac 2.0.8.3 manual:
+> Because of how these games were programmed, adding this feature would be quite troublesome, so currently there's no direct solution to this.  
+
+However, EoSD now supports this functionality, albeit with quirks. 
+
+Note: replays saved mid-game do not return to the title screen after the save point.
+
+---
+
+### What does "Everlasting BGM" mean?
+This option prevents the background music (BGM) from resetting when you restart the game.
+
+---
+
+### What does "Coercive Reporting" mean? (Shoot the Bullet/Double Spoiler)
+This feature locks the camera onto the boss at all times and eliminates the camera's recharge time.
+
+---
+
+### How do I switch languages?
+- The launcher automatically selects its language based on your Windows settings.  
+- To change the language in-game, use the shortcuts:  
+  - **ALT + 1**: Language 1  
+  - **ALT + 2**: Language 2  
+  - **ALT + 3**: Language 3  
+
+These shortcuts do not work in the launcher itself. Refer to the video in the **"How to Use"** section for detailed instructions.
+
+---
+
+## Bug Reporting
+
+### Where can I report bugs?
+Visit the [Issues tab on GitHub](https://github.com/touhouworldcup/thprac/issues) to report bugs or suggest improvements.
 
-# How to build:
-A Visual Studio solution is bundled that uses the Visual Studio 2022 build tools (v143) for building. However, thprac will compile perfectly fine with Visual Studio 2019 build tools (v142) as well. Your builds will support Windows XP perfectly fine, v141_xp build tools are not needed.
```

Comment:
thanks for removing that line. That line has been out of date for a while (I switched to v143 so I could use C++20)